i18n/locales/ja/ch-applications/qaoa.ipynb

```diff
@@ -7043,14 +7043,14 @@
     "a_beta          = np.arange(0, np.pi, step_size)\n",
     "a_gamma, a_beta = np.meshgrid(a_gamma,a_beta)\n",
     "\n",
-    "F1 = 3-(np.sin(2*a_beta)**2*np.sin(2*a_gamma)**2-0.5*np.sin(4*a_beta)*np.sin(4*a_gamma))*(1+np.cos(4*a_gamma)**2)\n",
+    "F1 = 3-(2*(np.sin(a_gamma))**2+np.sin(2*a_gamma)**2)*(np.sin(2*a_beta))**2/2+(10*np.sin(2*a_gamma)+np.sin(4*a_gamma))*np.sin(4*a_beta)/8\n",
     "\n",
     "# Grid search for the minimizing variables\n",
     "result = np.where(F1 == np.amax(F1))\n",
     "a      = list(zip(result[0],result[1]))[0]\n",
     "\n",
-    "gamma  = a[0]*step_size;\n",
-    "beta   = a[1]*step_size;\n",
+    "gamma  = a[1]*step_size;\n",
+    "beta   = a[0]*step_size;\n",
```

Comment:
Since the default indexing is Cartesian 'xy', the (gamma, beta) that gives the maximum value should be specified by (a[1],a[0])